components/brave_vpn/brave_vpn_service.cc

```diff
@@ -193,10 +193,11 @@ void BraveVpnService::OnGetSubscriberCredential(
     const std::string& body,
     const base::flat_map<std::string, std::string>& headers) {
   std::string subscriber_credential;
-
   bool success = status == 200;
   if (success) {
     subscriber_credential = GetSubscriberCredentialFromJson(body);
+  } else {
+    LOG(ERROR) << "Response from API was not HTTP 200. Body: " << body;
```

Comment:
If there's a risk that this could include some personal data / credentials, it might be better to `VLOG` the status code only instead of the body.

Reply:
Fixed with https://github.com/brave/brave-core/pull/10847/commits/4fb49af75b38a477ef6eb691bc256f5b87ecfab0 😄